hardware/Netatmo.h

```diff
@@ -2,29 +2,49 @@
 
 #include "DomoticzHardware.h"
 #include "../main/BaroForecastCalculator.h"
+#include <mutex>
```

Comment:
This should not be needed at it is already included in the precompiled header file (stdafx.h)